Formula/gmic.rb

```diff
@@ -0,0 +1,27 @@
+class Gmic < Formula
+  desc "Full-Featured Open-Source Framework for Image Processing"
+  homepage "https://gmic.eu/"
+  url "https://gmic.eu/files/source/gmic_1.7.8.tar.gz"
+  sha256 "3a2f32d79714239cfa56ecb10799b7d73362b4e2a852444bc24866272cd041a4"
+
+  depends_on "cmake" => :build
+  depends_on "jpeg" => :recommended
+  depends_on "libpng" => :recommended
+  depends_on "fftw" => :recommended
+  depends_on "opencv" => :optional
```

Comment:
This is from the Science tap, so it should be `depends_on "homebrew/science/opencv" => :optional`